pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -51,6 +53,8 @@ const (
 	kubeProxyServiceName = "kube-proxy"
 	// kubeletServiceName is the name of the kubelet Windows service
 	kubeletServiceName = "kubelet"
+	// windowsExporterServiceName is the name of the windows_exporter Windows service
+	windowsExporterServiceName = "windows_exporter"
```

Comment:
Is there a convention for Windows service naming? Please look at existing services.

Reply:
I think the prometheus community decided on the "windows_exporter" name so I don't think we can change this (though I know the _ does clash with all the other hyphenated services)

Reply:
We can name the service whatever we want, however I am fine if you want to stay with `windows_exporter`.

Reply:
Sure, let's stick with `windows_exporter` then.